<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OknoZerowanieUrzadzenia</class>
 <widget class="QDialog" name="OknoZerowanieUrzadzenia">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Czujki Liniowe - zerowanie urządzenia</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QFrame" name="frame_nadajnik">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="pierwszy">
        <property name="text">
         <string>Nadajnik</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="rb_pierwwszy_pion">
        <property name="text">
         <string>oś pionowa</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="rb_pierwszy_poziom">
        <property name="text">
         <string>oś pozioma</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QFrame" name="frame_odbiornik">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="drugi">
        <property name="text">
         <string>Odbiornik</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="rb_drugi_poziom">
        <property name="text">
         <string>oś pionowa</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="rb_drugi_pion">
        <property name="text">
         <string>oś pozioma</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QFrame" name="frame_filtry">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Filtry</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="rb_a">
        <property name="text">
         <string>A</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="rb_b">
        <property name="text">
         <string>B</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="rb_c">
        <property name="text">
         <string>C</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="3">
    <widget class="QFrame" name="frame_wozek">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Wózek</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="wozek_pion">
        <property name="text">
         <string>oś pionowa</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="wozek_poziom">
        <property name="text">
         <string>oś pozioma</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>27</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QFrame" name="frTitle">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Trwa zerowanie urządzenia</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
